src/coreclr/jit/llvm.cpp

```diff
@@ -1832,6 +1862,13 @@ void Llvm::buildStoreObj(GenTreeObj* storeOp)
     }
 }
 
+bool Llvm::isIndependentPromotedStrucField(LclVarDsc* varDsc)
+{
+    // TODO-LLVM: what happens when lvParentLcl == 0
```

Comment:
`0` is a valid local number (the first argument).

There is a compiler function which checks what we need already: `lvaIsFieldOfDependentlyPromotedStruct` (we don't really need to query for independent promotion).